### PR TITLE
Added a task that checks a file exists and is committed

### PR DESCRIPTION
**Pull request by super-secret-github-user**

### CODE REVIEW

File: src/kraken/std/git/tasks/important_file_check_task.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class ImportantFileCheckTask(Task):
+    file_to_check: Property[str]
+
+    def file_committed(self):
+        command = ["git", "ls-files", "--", self.file_to_check.get()]
+        return sp.getoutput(command)
+
+    def execute(self) -> TaskStatus:
+        if not exists(self.file_to_check.get()):
```
Comment (tetigi) [inline src/kraken/std/git/tasks/important_file_check_task.py:17]: Do we have to ensure that the `file_to_check` is fully qualified? Is it relative to this project?

File: src/kraken/std/git/tasks/important_file_check_task.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+import subprocess as sp
+from os.path import exists
+
+from kraken.core.api import Property, Task, TaskStatus
+
+
+class ImportantFileCheckTask(Task):
+    file_to_check: Property[str]
```
Comment (tetigi) [inline src/kraken/std/git/tasks/important_file_check_task.py:10]: Let's use `pathlib.Path` here.

**Comment by tetigi** [comment]
@super-secret-github-user - the reason this fails is because it's not ran _from_ this repo, it's a fork (I forgot to check this whilst we were chatting). Change the `remote` on your local to point to this and push it again, then remake this MR and you should be good to go :)

**Comment by NiklasRosenstein** [comment]
I presume this is to have Kraken barf when the lock file is missing?

**Pull request closed without merging**